tests/network-tests/src/fixtures/membership/InviteMembersHappyCaseFixture.ts

```diff
@@ -61,7 +61,7 @@ export class InviteMembersHappyCaseFixture extends StandardizedFixture {
         handle,
         rootAccount,
         controllerAccount,
-        metadata: { name, about, avatar, externalResources, validatorAccount },
+        metadata: { name, about, avatar, externalResources },
```

Comment:
Please reverse the changes on this file from this PR. This test has nothing to do with this PR (it's about #4915)